contents/product-engineers/behavioral-analytics.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: How to use behavioral analytics as an engineer
+date: 2024-02-12
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+tags:
+ - Guides
+ - Product analytics
+ - Product
+crosspost:
+  - Blog
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import ReplayLight from '../images/product-engineers/behavioral-analytics/replay.png'
+import ReplayDark from '../images/product-engineers/behavioral-analytics/replay-dark.png'
+
+Back in the day, companies could see how people were interacting with their products. They could talk to them, knock on their door, and give them a firm handshake. Now, many companies have no direct interactions with their users. Instead, they rely on behavioral analytics. 
+
+Behavioral analytics is the data about the usage of your site or app. It is a combination of a user and an action. Looking at this data, both individually and in aggregate, enables you to monitor the experience you are providing, identify popular features, and discover areas needing improvement.
+
+As an engineer, you might not think of this data as important to you, but you should. An app is shaped by its users, whether they are external or internal. Everything you build is in service to better help users (at some point), and analytics is core to understanding what they want. To help you do this, this post goes over how to get started and use behavioral analytics as an engineer. 
```

Comment:
Take a second go at this intro. Things to consider:

- Why should the reader listen to us? I think it's worth talking about our credentials (we build analytics tools for engineers, we talk to them and support them in our community etc.). We've gone through this experience ourselves. I think credibility is important for what comes later.

- Can we start with an opinion? Grab the reader from the start.